dlk/python/dlk/plugins/tf.py

```diff
@@ -924,13 +924,13 @@ def infer_dtype() -> DataType:
                 input_ops,
                 dimension_format=current_format,
             )
-        elif op_type == 'QTZ_binary_channel_wise_mean_scaling':
+        elif op_type == 'QuantizeBinaryChannelWiseMeanScaling':
```

Comment:
It would be better to accept old name `QTZ_binary_channel_wise_mean_scaling` here, otherwise old protocol buffer file will be failed to convert.

Reply:
I decided to add a function to convert this deprecated operator here for backward compatibility. What do you think?

https://github.com/blue-oil/blueoil/pull/659/files#diff-15fb8a5fce9a3d19dd944a3764c512bdR394-R401

Reply:
good idea!